client/LICENSE

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
+                       Version 3, 29 June 2007
```

Comment:
probably copy/paste, let's not have a second license file in here, let's remove this file.